events/api.py

```diff
@@ -2268,6 +2263,24 @@ def _filter_event_queryset(queryset, params, srs=None):
         elif val.lower() == 'false':
             queryset = queryset.exclude(offers__is_free=True)
 
+    val = params.get('suitable_for', None)
+    ''' Excludes all the events that have max age limit below or min age limit above the age or age range specified.
+    Suitable events with just one age boundary specified are returned, events with no age limits specified are
+    excluded. '''
+
+    if val:
+        vals = val.split(',')
+        if len(vals) > 2:
+            raise ParseError(f'suitable_for takes maximum two values, you provided {len(vals)}')
+        int_vals = [validate_digit(i, 'suitable_for') for i in vals]
+        if len(int_vals) == 2:
+            lower_boundary = min(int_vals)
+            upper_boundary = max(int_vals)
+        else:
+            lower_boundary = upper_boundary = int_vals[0]
```

Comment:
I'm not sure if this is any clearer, but it sure is smaller:
`lower_boundary, upper_boundary = sorted([validate_digit(i, 'suitable_for') for i in vals]) if len(vals)==2 else [*vals,None]`

Reply:
it is smaller indeed, but I think it is less clear so I'd stick to the original version.